src/style.css

```diff
@@ -2,0 +1,96 @@
+@font-face {
+  font-family: Avenir, sans-serif;
+  src: url(/src/fonts/Avenir.ttc) format('truetype');
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: 'Avenir Book', sans-serif;
+  src: url(/src/fonts/Avenir-Book.ttf);
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: 'Avenir Heavy', sans-serif;
+  src: url(/src/fonts/Avenir-Heavy.ttf) format('truetype');
+  font-weight: 800;
+}
+
+body {
+  margin: 0;
+  box-sizing: border-box;
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+}
+
+.bar {
+  background-image: url(/src/images/Search.svg);
+  background-repeat: no-repeat;
+
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-sizing: border-box;
+  align-items: center;
+
+  display: inline-block;
+}
+
+.big-bar {
+  font-family: Avenir, sans-serif;
+  background-position: 26px 25px;
+  background-size: 19px;
+
+  padding-left: 62px;
+  margin: 20px 0;
+
+  height: 70px;
+  width: 100vw;
+}
+
+.small-bar {
+  font-family: 'Avenir Book', sans-serif;
```

Comment:
Ensure that the `font-family` used here matches the custom font name defined in the `@font-face` rule, without including fallback fonts.